tasty-plutus/src/Test/Tasty/Plutus/Script/Unit.hs

```diff
@@ -194,6 +200,22 @@ shouldValidateTracing name f td cb = case td of
     tt <- asks (singleTest name . Minter Pass (Just f) td cb)
     tell . Seq.singleton $ tt
 
+{- | Specify that, given this test data and context, the validation should
+ succeed. All context outputs are converted to a list of inputs,
```

Comment:
The comma in that sentence is not necessary. A better phrasing would be:

"All context outputs are converted to a list of inputs: these can be used to build new contexts."